polars/polars-core/src/chunked_array/logical/date.rs

```diff
@@ -40,6 +41,12 @@ impl LogicalType for DateChunked {
                     .into_datetime(*tu, tz.clone())
                     .into_series())
             }
+            (Date, Time) => Ok(Int64Chunked::from_iter_values(
```

Comment:
We can use `Int64Chunked::full(self.name(), 0, self.len())` here.

Reply:
Hah; I should have known there was a super-optimal path ;)
Will update accordingly.